auraescript/macros/src/ops.rs

```diff
@@ -78,17 +78,30 @@ pub(crate) fn ops_generator(input: TokenStream) -> TokenStream {
                     let output_type = Ident::new(output_type, file_path_span);
                     let name = Ident::new(&m.name().to_snake_case(), file_path_span);
 
+                    // Magic OpState from deno (https://github.com/denoland/deno/blob/b6ac54815c1bcfa44a45b3f2c1c982829482477f/ops/lib.rs#L295)
                     quote! {
                         #[::deno_core::op]
                         pub(crate) async fn #op_ident(
-                            req: ::proto::#module::#input_type
+                            opstate: Rc<RefCell<OpState>>, // Auto filled by deno macro, call from typescript ignoring this parameter
```

Comment:
Nit: `op_state`

Like the other comment, but this one is less bothersome since my linter doesn't care about it, lol.